cedar-policy/src/api.rs

```diff
@@ -1232,6 +1463,13 @@ pub struct EntityUid(ast::EntityUID);
 
 impl EntityUid {
     /// Returns the portion of the Euid that represents namespace and entity type
+    /// ```
+    /// use cedar_policy::{Entity, EntityId, EntityTypeName, EntityUid};
+    /// use std::str::FromStr;
```

Comment:
let's hide these imports
```suggestion
    /// # use cedar_policy::{Entity, EntityId, EntityTypeName, EntityUid};
    /// # use std::str::FromStr;
```